libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -289,7 +289,7 @@ float AP_Baro::get_altitude_difference(float base_pressure, float pressure) cons
 float AP_Baro::get_EAS2TAS(void)
 {
     float altitude = get_altitude();
-    if ((fabsf(altitude - _last_altitude_EAS2TAS) < 100.0f) && !is_zero(_EAS2TAS)) {
+    if ((fabsf(altitude - _last_altitude_EAS2TAS) < 25.0f) && !is_zero(_EAS2TAS)) {
```

Comment:
I can make it a define, but in this case it doesn't feel like a particularly useful one. Not opposed if the consensus is that would be more readable, but since this is very readable as is and we don't use it any where else I'm not sure it would make it that much clearer.